File: wastelessapp/src/app/models/grocery-model.ts
import {GroceryList} from "./grocery-list";
import {Item} from "./item";

export class GroceryModel {
  GrListObj:GroceryList;
  ItList:any;

  constructor(gl:GroceryList,it:any) {
    this.GrListObj = new GroceryList(gl.GroceryListName,gl.GroceryListId);
    this.ItList = it;
  }
}

File: WasteLESS/WasteLESS/Models/GroceryModel.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace WasteLESS.Models
{
    public class GroceryModel
    {

        public GroceryList GrListObj { get; set; }
        public IEnumerable<Item> ItList { get; set; }
        
    }
}
